test/performance/odp_timer_perf.c

```diff
@@ -553,6 +621,143 @@ static int test_worker(void *arg)
 	return ret;
 }
 
+static int set_cancel_mode_worker(void *arg)
+{
+	uint64_t tick, start_tick, period_tick, nsec;
+	uint64_t c1, c2, diff;
+	int thr, status;
+	uint32_t i, j, worker_idx;
+	odp_event_t ev;
+	odp_time_t t1, t2;
+	odp_timer_t timer;
+	odp_timer_pool_t tp;
+	odp_timeout_t tmo;
+	thread_arg_t *thread_arg = arg;
+	test_global_t *global = thread_arg->global;
+	test_options_t *test_options = &global->test_options;
+	uint32_t num_tp = test_options->num_tp;
+	uint32_t num_timer = test_options->num_timer;
+	uint32_t num_worker = test_options->num_cpu;
+	int ret = 0;
+	int started = 0;
+	uint64_t test_rounds = test_options->test_rounds;
+	uint64_t num_tmo = 0;
+	uint64_t num_cancel = 0;
+	uint64_t num_set = 0;
+
+	thr = odp_thread_id();
+	worker_idx = thread_arg->worker_idx;
+	t1 = ODP_TIME_NULL;
+	c1 = 0;
+
+	/* Start all workers at the same time */
+	odp_barrier_wait(&global->barrier);
+
+	while (1) {
+		ev = odp_schedule(NULL, ODP_SCHED_NO_WAIT);
+
+		if (odp_unlikely(ev != ODP_EVENT_INVALID)) {
+			/* Timeout, set timer again. When start_tick is large enough, this should
+			 * not happen. */
+			timer_ctx_t *ctx;
+
+			tmo   = odp_timeout_from_event(ev);
+			ctx   = odp_timeout_user_ptr(tmo);
+			i     = ctx->tp_idx;
+			j     = ctx->timer_idx;
+			timer = global->timer[i][j];
+			start_tick  = global->timer_pool[i].start_tick;
+			period_tick = global->timer_pool[i].period_tick;
+			tick = start_tick + j * period_tick;
+
+			status = odp_timer_set_rel(timer, tick, &ev);
+			num_tmo++;
+			num_set++;
+
+			if (status != ODP_TIMER_SUCCESS) {
+				printf("Error: Timer set (tmo) failed (ret %i)\n", status);
```

Comment:
While you're at it, error prints could be updated to use `ODPH_ERR()` macro, but this is not mandatory.

Reply:
Fixed in v6 for all error prints in the file